DigitalLearningSolutions.Web/Helpers/SelectListHelper.cs

```diff
@@ -20,5 +20,15 @@ public static IEnumerable<SelectListItem> MapOptionsToSelectListItems(IEnumerabl
         {
             return options.Select(o => new SelectListItem(o, o, o == selected)).ToList();
         }
+
+        public static IEnumerable<SelectListItem> MapOptionsToSelectListItems(IEnumerable<(string, string)> options)
+        {
+            return options.Select(NewSelectListItemFromTupleString);

```

Comment:
Could we just do:
`return options.Select(option => new SelectListItem(option.text, option.value));`

(I don't think we get much benefit from the NewSelectListItemFromTupleString but can be convinced otherwise :P

Reply:
I think it was more of a left over as I was refactoring. Changed to just do it all in the public one.